content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -588,3 +597,299 @@ As part of the Peruvian localization, besides creating credit notes from an exis
+
+New required information
+************************
+
+Version 2.0 requires additional information on the general configuration, vehicles, contacts, and
+products. In the general configurations, it is now necessary to add new credentials that are
+acquired in the |SUNAT| portal.
+
+Flow
+****
+
+.. image:: peru/peru-flow-remision.png
+   :align: center
+   :alt: Flow for Guia de Remision emission.
+
+#. Generate sales order
+#. Validate delivery order
+#. Generate waybill
+#. Perform validation or rejection
+#. Send CDR
+#. Receive validated CDR on outbound order
+#. Carrier travels with printed |GRE|
+#. Deliver goods
+
+Cancellations
+*************
+
+Currently, the |SUNAT| no longer uses the term "Anula", but now uses the term "Dar de baja" for
+cancellations. Therefore, **both** the sender and the carrier can cancel the electronic waybill, so
+long as the following conditions are met:
+
+- The shipment has not been initiated.
+- If the shipment has been initiated, the receiver **must** be changed before reaching the final
+  destination.
+
+Testing
+*******
+
+The |SUNAT| does not support a test environment. This means that any delivery guides that were
+generated by mistake **will** be sent to the |SUNAT|.
+
+If, by mistake, the waybill was created in this environment, it is necessary to delete it from the
+|SUNAT| portal.
+
+Configuration
+~~~~~~~~~~~~~
+
+.. important::
+   - Electronic sender's |GRE| is currently the only supported type of waybill in Odoo.
+   - The delivery guide is dependent on the Odoo *Inventory* app, the :guilabel:`l10n_pe_edi` and
+     :guilabel:`l10n_pe` modules.
+   - A second user **must** be added for the creation of electronic documents.
+
+After following the steps to configure the :ref:`electronic invoicing <peru/accounting-settings>`
+and the :ref:`master data <peru/master_data>`, navigate to :menuselection:`Apps` and clear any
+default filters set. Search for the `l10n_pe_edi_stock_20` module and click the :guilabel:`Activate`
+button.
+
+Next, you will need to obtain the *client ID* and *client secret* from |SUNAT|. To do so, follow the
+`manual de servicios web plataforma nueva GRE
+<https://cpe.sunat.gob.pe/sites/default/files/inline-files/Manual_Servicios_GRE.pdf>`_.
+
+.. note::
+   In the |SUNAT| portal, it is important to have the correct access rights enabled, as they may
+   differ from the user set for electronic invoicing.
+
+These credentials should be used to configure the delivery guide general settings from
+:menuselection:`Accounting --> Configuration --> Settings --> Peruvian Electronic Invoicing`.
+
+.. note::
+   It is required to follow the format `RUC + UsuarioSol` (e.g `20557912879SOLUSER`) for the
+   :guilabel:`Guide SOL User` field, depending on the user selected when generating the |GRE| API
+   credentials in the |SUNAT| portal.
+
+.. image:: peru/gre-fields-example.png
+   :align: center
+   :alt: Example for the SUNAT Delivery Guide API section configuration.
```

Comment:
to which paragraph does this refer to? is this under ':menuselection:`Accounting --> Configuration --> Settings --> Peruvian Electronic Invoicing`.' if so then disregard my previous comment and I'd put the explaining paragraph related to the screenshot closer.